setup.py

```diff
@@ -866,8 +868,8 @@ class SaltDistribution(distutils.dist.Distribution):
          'Salt\'s pre-configured SPM formulas directory'),
         ('salt-spm-pillar-dir=', None,
          'Salt\'s pre-configured SPM pillar directory'),
-        ('salt-spm-reactor-dir=', None,
-         'Salt\'s pre-configured SPM reactor directory'),
```

Comment:
Any reason to remove the `salt-spm-reactor-dir`?

Reply:
Looks like a cut that should have been a copy.